### ISSUE TITLE
[Bug][a11y]: Table selected row state with forced-colors: active 

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3400]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Table

### Expected behavior

It should be possible to distinguish between the selected or not-selected state of a row in Windows High Contrast Mode/`forced-colors: active`.

### Actual behavior

With `forced-colors: active`/Windows High Contrast Mode, it is impossible to distinguish between the selected and not selected state when `selects` is `undefined`.

Test https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/table--no-selects-specified with `force-colors: active`

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by jawinn** [comment]
On the Spectrum CSS side, this one has been addressed in the beta of the token migration for table. It will need to be tested and confirmed on the SWC end in the integration that's currently being worked on. 

It's using the system color pair `SelectedItem` and `SelectedItemText` through an `@supports`, with the default fallback to `Highlight` and `HighlightText`.

![Screenshot 2023-08-10 at 3 59 04 PM](https://github.com/adobe/spectrum-web-components/assets/965114/359f423e-5e9b-48fd-91ad-7de0b3e935ab)

![Screenshot 2023-08-10 at 3 59 41 PM](https://github.com/adobe/spectrum-web-components/assets/965114/12c4fe65-c6c0-4db3-927d-8f304a176146)
